src/streamlink/plugins/rtbf.py

```diff
@@ -1,8 +1,10 @@
 import datetime
 try:
     from HTMLParser import HTMLParser
+    html_unescape = unescape = HTMLParser().unescape
 except ImportError:
-    from html.parser import HTMLParser
+    from html import unescape as html_unescape
```

Comment:
this is actually in the wrong file it should be in **compat.py** and py3 before py2

```py
try:
    from html import unescape as html_unescape
except ImportError:
    from HTMLParser import HTMLParser
    html_unescape = HTMLParser().unescape
```

Reply:
Could move it to compat, I just left it where it was.

Reply:
What do you mean py3 before py2?

Reply:
https://docs.python.org/3/library/html.html#html.unescape

`html` is python3 so I would do it like this

```py
try:
    # PYTHON 3
except ImportError:
    # PYTHON 2
```

Reply:
True, brainfart

Reply:
Well, HTMLParser isn't in 3 anywy :)